CONTRIBUTING.md

```diff
@@ -335,6 +343,9 @@ Lastly, if the feature really is a game changer or you're very proud of it, cons
     cd auto-sklearn
     git checkout -b my_new_branch development
 
+    # Initaize autosklearn/automl_common submodule
```

Comment:
```suggestion
    # Initialize autosklearn/automl_common submodule
```

Reply:
I should honestly consider having a spell checker on when looking at `.md` files and `.rst` files ...

Reply:
https://github.com/marketplace/actions/github-spellcheck ;)